### DISCUSSION TITLE
Welcome to Computer-Networks-Mini-Projects Discussions!

### DISCUSSION THREAD

**Discussion by anzal1** [anzal1/Computer-Networks-Mini-Projects#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anzal1** [comment]
#1 is completly in java , is it cross-platfrom or generic.

**Reply by Faizullah9181** [reply]
!!yes it is cross platform
